examples/filters/colorfilter/color_filter.go

```diff
@@ -0,0 +1,152 @@
+// Package colorfilter implements a modular camera that filters the output of an underlying camera and only keeps
+// captured data if the vision service detects a certain color in the captured image.
+package colorfilter
+
+import (
+	"context"
+	"fmt"
+	"image"
+	"time"
+
+	"github.com/edaniels/golog"
+	"github.com/pkg/errors"
+	"github.com/viamrobotics/gostream"
+
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/data"
+	"go.viam.com/rdk/pointcloud"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/rimage/transform"
+	"go.viam.com/rdk/services/vision"
+)
+
+var (
+	Model            = resource.NewModel("example", "filter", "colorfilter")
```

Comment:
I think if I recall correctly these model names don't have a set scheme, but still I'm wondering if it would be better to name this something that indicates its a camera model. Maybe it doesn't matter or we can revisit later.

Reply:
True - I'll connect with Naomi to see if they already have a scheme for tutorial model names, and if not can discuss what makes the most sense for this!